clang/lib/APINotes/APINotesWriter.cpp

```diff
@@ -33,6 +33,7 @@ class APINotesWriter::Implementation {
   /// Mapping from strings to identifier IDs.
   llvm::StringMap<IdentifierID> IdentifierIDs;
 
+public:
```

Comment:
Why make all this public?

Reply:
Reverted this part of the change by making `APINotesWriter` a friend class of `Implementation`